scripts/eval/yamls/eval_gauntlet_v0.1.yaml

```diff
@@ -0,0 +1,200 @@
+eval_gauntlet:
+  weighting: EQUAL
+  subtract_random_baseline: true
+  rescale_accuracy: true
+  averages:
+    core_average:
+    - world_knowledge
+    - commonsense_reasoning
+    - language_understanding
+    - symbolic_problem_solving
+    - reading_comprehension
+    # - programming
+  categories:
+  - name: world_knowledge
+    benchmarks:
+    - name: jeopardy
+      num_fewshot: 10
```

Comment:
Do we wanna keep these at 10? If we'd like to reduce them, now might be the time

Reply:
➕

Reply:
@mansheej what do you think?

Reply:
@codestar12  recommended leaving for now and changing later